Mathlib/Algebra/Homology/ShortComplex/Ab.lean

```diff
@@ -134,7 +137,18 @@ lemma ab_exact_iff_range_eq_ker : S.Exact ↔ S.f.hom.range = S.g.hom.ker := by
   · intro h
     rw [h]
 
-variable {S}
+alias ⟨Exact.ab_range_eq_ker, _⟩ := ab_exact_iff_range_eq_ker
+
+/-- In an exact complex, an abelian group sandwiched between two finite abelian groups is finite. -/
+lemma Exact.finite_ab {S : ShortComplex Ab.{u}} (hS : S.Exact) [Finite S.X₁] [Finite S.X₃] :
```

Comment:
```suggestion
lemma Exact.ab_finite {S : ShortComplex Ab.{u}} (hS : S.Exact) [Finite S.X₁] [Finite S.X₃] :
```
The local naming convention seems to be to prepend `ab` at every lemma about `ShortComplex Ab` so maybe we should follow it here?